app/src/main/java/free/rm/skytube/gui/activities/YouTubePlayerActivity.java

```diff
@@ -20,8 +20,12 @@
 import android.app.Activity;
 import android.content.pm.ActivityInfo;
 import android.os.Bundle;
+<<<<<<< HEAD
+import android.util.Log;
+=======
```

Comment:
You merged something incorrectly

Reply:
It can be because of old messed up commits and I tried to clean but it is still there. Maybe ram-on can fix it when he adds these changes.